bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -245,13 +249,108 @@ assert len(args.log_param) == len(args.fit_param) == len(args.smoothing_width)
 
 init_logging(args.verbose)
 
-fits = h5py.File(args.template_fit_file, 'r')
+analysis_time = 0
+attr_dict = {}
+
+tid = numpy.array([], dtype=int)
+nabove = numpy.array([], dtype=int)
+ntotal = numpy.array([], dtype=int)
+alpha = numpy.array([], dtype=float)
+median_sigma = numpy.array([], dtype=float)
+
+logging.info("Loading input template fits")
+for filename in args.template_fit_file:
+    with h5py.File(filename, 'r') as fits:
+        # get attributes from the template-level fit
+        for k in fits.attrs.keys():
+            if k == 'analysis_time':
+                # For this, we want the mean, not check that it is identical
+                analysis_time += fits.attrs['analysis_time']
+                continue
+            if k not in attr_dict:
+                # This is the first time this attribute
+                # has been encountered
+                attr_dict[k] = fits.attrs[k]
+            elif not attr_dict[k] == fits.attrs[k]:
+                # Check that the files are consistent with one another
+                err_msg = f"Input files are not consistent, file {filename} "
+                err_msg += f"has attribute {k} of {fits.attrs[k]}, whereas "
+                err_msg += f"previous files have value {attr_dict[k]}"
+                raise RuntimeError(err_msg)
+
+        # get template id and template parameter values
```

Comment:
I guess it will be clearer to see what happens in the next step if you say explicitly that these end up being big arrays each of which contains n_templates * n_files entries